Python/codecs.c

```diff
@@ -50,6 +50,26 @@ int PyCodec_Register(PyObject *search_function)
     return -1;
 }
 
+PyObject *
```

Comment:
For the C function, I would prefer to return 0 on success and -1 on error.

Reply:
done.